src/api/wayfire/view-access-interface.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "wayfire/condition/access_interface.hpp"
+#include "wayfire/view.hpp"
+#include <string>
+#include <tuple>
+
+namespace wf
+{
+
+/**
+ * @brief The view_access_interface_t class is a view specific implementation of access_interface_t.
+ *
+ * Refer to the docs of access_interface_t for more information.
+ *
+ * The following properties are supported:
+ *
+ * format:
+ * property -> type (comment)
+ *
+ * "app_id" -> std::string
+ * "title" -> std::string
+ * "role" -> std::string
+ * "fullscreen" -> bool
+ * "activated" -> bool
+ * "minimized" -> bool
+ * "tiled" -> bool
+ * "maximized" -> bool
+ * "floating" -> bool
```

Comment:
Hmm, seems like there is no difference between `tiled == true` and `floating == false`, is this correct? In that case, I see no benefit in having "floating".